sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -80,16 +77,42 @@ public ConfigurationClient(Uri endpoint, TokenCredential credential, Configurati
 
             _endpoint = endpoint;
             _syncTokenPolicy = new SyncTokenPolicy();
+            _syncToken = _syncTokenPolicy.ToString();
             _pipeline = CreatePipeline(options, new BearerTokenAuthenticationPolicy(credential, GetDefaultScope(endpoint)), _syncTokenPolicy);
+            _apiVersion = options.Version;
+
+            ClientDiagnostics = new ClientDiagnostics(options);
```

Comment:
Pass `suppressNestedClientActivities` as true in `new ClientDiagnostics(options, true);` to disable nested scopes.